bot.py

```diff
@@ -35,8 +36,373 @@ async def challenge(ctx: commands.Context, opponent: str):
+					}],
+					"Friendship": 0,
+					"OriginalTrainerID": 0,
+					"Type": 0
+				}
+			}],
+			"Targets": [{
+				"Team": 0,
+				"Party": 0,
+				"Slot": 0,
+				"Pokemon": {
+					"Name": "Mew",
+					"NatDex": 151,
+					"Level": 6,
+					"TotalExperience": 179,
+					"Gender": 0,
+					"IVs": [3, 1, 1, 4, 0, 3],
+					"EVs": [7, 24, 41, 11, 3, 20],
+					"Nature": 5,
+					"Stats": [28, 17, 17, 17, 17, 17],
+					"StatModifiers": [0, 0, 0, 0, 0, 0, 0, 0, 0],
+					"StatusEffects": 0,
+					"CurrentHP": 26,
+					"Moves": [{
+						"ID": 447,
+						"Name": "Grass Knot",
+						"Type": 2048,
+						"Category": 2,
+						"Targets": 10,
+						"CurrentPP": 20,
+						"MaxPP": 20,
+						"Priority": 0,
+						"Power": 0,
+						"Accuracy": 100
+					}, {
+						"ID": 449,
+						"Name": "Judgment",
+						"Type": 1,
+						"Category": 2,
+						"Targets": 10,
+						"CurrentPP": 10,
+						"MaxPP": 10,
+						"Priority": 0,
+						"Power": 100,
+						"Accuracy": 100
+					}, {
+						"ID": 322,
+						"Name": "Cosmic Power",
+						"Type": 8192,
+						"Category": 0,
+						"Targets": 7,
+						"CurrentPP": 20,
+						"MaxPP": 20,
+						"Priority": 0,
+						"Power": 0,
+						"Accuracy": 0
+					}, {
+						"ID": 280,
+						"Name": "Brick Break",
+						"Type": 2,
+						"Category": 1,
+						"Targets": 10,
+						"CurrentPP": 15,
+						"MaxPP": 15,
+						"Priority": 0,
+						"Power": 75,
+						"Accuracy": 100
+					}],
+					"Friendship": 0,
+					"OriginalTrainerID": 0,
+					"Type": 0
+				}
+			}, {
+				"Team": 1,
+				"Party": 1,
+				"Slot": 0,
+				"Pokemon": {
+					"Name": "Absol",
+					"NatDex": 359,
+					"Level": 60,
+					"TotalExperience": 211060,
+					"Gender": 0,
+					"IVs": [4, 2, 3, 3, 1, 1],
+					"EVs": [5, 31, 11, 41, 29, 40],
+					"Nature": 2,
+					"Stats": [151, 166, 80, 102, 81, 101],
+					"StatModifiers": [0, 0, 0, 0, 0, 0, 0, 0, 0],
+					"StatusEffects": 0,
+					"CurrentHP": 151,
+					"Moves": [{
+						"ID": 230,
+						"Name": "Sweet Scent",
+						"Type": 1,
+						"Category": 0,
+						"Targets": 11,
+						"CurrentPP": 20,
+						"MaxPP": 20,
+						"Priority": 0,
+						"Power": 0,
+						"Accuracy": 100
+					}, {
+						"ID": 429,
+						"Name": "Mirror Shot",
+						"Type": 256,
+						"Category": 2,
+						"Targets": 10,
+						"CurrentPP": 10,
+						"MaxPP": 10,
+						"Priority": 0,
+						"Power": 65,
+						"Accuracy": 85
+					}, {
+						"ID": 284,
+						"Name": "Eruption",
+						"Type": 512,
+						"Category": 2,
+						"Targets": 11,
+						"CurrentPP": 5,
+						"MaxPP": 5,
+						"Priority": 0,
+						"Power": 150,
+						"Accuracy": 100
+					}, {
+						"ID": 157,
+						"Name": "Rock Slide",
+						"Type": 32,
+						"Category": 1,
+						"Targets": 11,
+						"CurrentPP": 10,
+						"MaxPP": 10,
+						"Priority": 0,
+						"Power": 75,
+						"Accuracy": 90
+					}],
+					"Friendship": 0,
+					"OriginalTrainerID": 0,
+					"Type": 0
+				}
+			}]
+		})
```

Comment:
Where is all this data coming from and what does it mean? Is this supposed to be placeholder data for now that's actually supposed to be coming from the API?

Reply:
Yes, it's placeholder.